lib/user_interface.rb

```diff
@@ -54,6 +62,14 @@ def ask_for_fields
     end
   end
 
+  def display_contact(contact_hash)
```

Comment:
I would avoid adding the type (`hash`) as part of the variable name. In the future, you may want to use a different type to store a contact, for example you may use an object rather than a hash.

```ruby
  def display_contact(contact)
```